service-deployment/cleanup-service.py

```diff
@@ -96,6 +101,15 @@ def main():
     error_msg = "Successfully to delete cleaning-one-shot"
     execute_shell(shell_cmd, error_msg)
 
+    serv = 'cluster-configuration'
+    shell_cmd = 'chmod a+x ./bootstrap/{0}/{1}'.format(serv, service_config['servicelist'][serv]['stopscript'])
+    error_msg = 'Failed make the {0} stop script executable'.format(serv)
+    execute_shell(shell_cmd, error_msg)
+
+    shell_cmd = './bootstrap/{0}/{1}'.format(serv, service_config['servicelist'][serv]['stopscript'])
+    error_msg = 'Failed stop the service {0}'.format(serv)
+    execute_shell(shell_cmd, error_msg)
+
```

Comment:
this could become a function: cleanOneService

Reply:
Resolved.